server/lib/sync-agent.js

```diff
@@ -377,6 +381,10 @@ export default class SyncAgent {
           hull_summary: _.get(err, "message", "Server Error: Encountered error during sync operation"),
           type: this.import_type
         });
+        this.hull.put("app/status", {
```

Comment:
What about reverting the status to ok if everything goes fine here ? 
... because I don't see where we would recover from an error status when it has been set

Reply:
We can do `put({ status: "ok" })` near incoming.job.success. Would it work?

Reply:
I guess so